src/store/modules/user/mutation-types.ts

```diff
@@ -4,4 +4,5 @@ export const USER_END_SESSION = SN_USER + '/END_SESSION'
 export const USER_INFO_UPDATED = SN_USER + '/INFO_UPDATED'
 export const USER_CURRENT_FACILITY_UPDATED = SN_USER + '/CURRENT_FACILITY_UPDATED'
 export const USER_INSTANCE_URL_UPDATED = SN_USER + '/INSTANCE_URL_UPDATED' 
-export const USER_SHIPPING_ORDERS_STATUS_UPDATED = SN_USER + '/SHIPPING_ORDERS_STATUS_UPDATED'
\ No newline at end of file
+export const USER_SHIPPING_ORDERS_STATUS_UPDATED = SN_USER + '/SHIPPING_ORDERS_STATUS_UPDATED'
+export const USER_CURRENT_STORE_UPDATED = SN_USER + '/CURRENT_STORE_UPDATED'
```

Comment:
```suggestion
export const USER_CURRENT_ECOM_STORE_UPDATED = SN_USER + '/CURRENT_ECOM_STORE_UPDATED'
```

Reply:
Changed code as per consistent approach of code.